tools/docker/syzbot/run-syz-command.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright 2024 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+set -e # exit on any problem
+set -o pipefail
+
+syzDir=$(mktemp -d)
+
+git clone https://github.com/google/syzkaller $syzDir
```

Comment:
Makes to at least do a shallow checkout since this is checked our each time and it thrown away.

Reply:
Done.